config/fortify-client.properties

```diff
@@ -0,0 +1 @@
+fortify.client.releaseId=65430
```

Comment:
release id is 65417. please pick the respective value for each repo.

Reply:
done